templates/dashboard.html

```diff
@@ -1,230 +1,87 @@
-                    <!-- Kelas Input Field -->
-                    <div class="mb-4 w-full">
-                        <input type="text" id="kelas4" class="bg-gray-50 border border-gray-300 shadow-md shadow-gray-500 text-gray-900 text-sm rounded-lg w-40 p-2.5 mb-10 mx-auto block text-center" placeholder="Kelompok" required />
-                    </div>
-
-                    <!-- Image Display -->
-                    <img src="/static/images/File.png" alt="Image 1" class="h-14 mb-4 mx-auto">
-
-                    <!-- File Upload Section -->
-                    <label for="file-upload-8" class="text-teal-600 font-semibold cursor-pointer">
-                        <input type="file" id="file-upload-8" class="hidden" onchange="displayFileName()" />
-                        Upload File
-                    </label>
-                    
-                    <!-- Display File Name -->
-                    <div id="file-name-8" class="mt-2 text-gray-500"></div>
+            <div class="button-wrapper">
+                <form action="/dashboard/result" method="post">
+                    <button id="compare-button"
+                        class="bg-gradient-to-r from-deep-blue to-light-blue text-white font-bold py-3 px-6 rounded-lg hover:from-teal-500 hover:to-teal-700 transition-all duration-300">
+                        COMPARE
+                    </button>
                 </form>
             </div>
-            <div class="mt-1 text-md font-semibold text-red-600" >Accepted file types : .c document</div>
+        </div>
+        {% else %}
+        <h1 class="text-center text-3xl font-bold text-deep-blue mt-6 mb-8">
+            Upload Your Files
+        </h1>
+        <form action="/upload" method="post" id="upload-form" enctype="multipart/form-data"
+            class="flex flex-col items-center justify-center mb-6">
+            <label for="multi-file-upload" class="text-teal-600 font-semibold cursor-pointer text-center">
+                <img src="/static/images/File.png" alt="Upload Icon" class="h-14 mx-auto mb-2" />
+                <span class="block text-lg font-bold text-deep-blue">
+                    Click to Select .C Files
+                </span>
+                <input type="file" id="multi-file-upload" name="files" class="hidden" accept=".c" multiple />
+            </label>
+        </form>
+
+        <div id="files-container" class="grid grid-cols-1 md:grid-cols-4 gap-6 mb-8">
 
-            <!-- Centered 'COMPARE' Button -->
-            <div class="col-span-1 md:col-span-4 flex justify-center mt-4">
-                <button type="submit" class="text-white bg-gradient-to-r from-deep-blue to-light-blue focus:ring-4 focus:outline-none focus:ring-blue-300 font-medium rounded-full text-sm w-40 px-5 py-2.5 text-center">
-                    COMPARE
-                </button>
-            </div>
         </div>
+        {% endif %}
     </section>
 
     <script src="/static/js/script.js"></script>
```

Comment:
please put all script in the head element for easier tracking, thanks!

Reply:
(same here too!)